engine/requirements.in

```diff
@@ -52,7 +52,7 @@ PyMySQL==1.1.1
 python-telegram-bot==13.13
 recurring-ical-events==2.1.0
 redis==5.0.1
-regex==2021.11.2
+regex==2024.7.24
```

Comment:
from the Python docs, TIL;

> See also The third-party [regex](https://pypi.org/project/regex/) module, which has an API compatible with the standard library [re](https://docs.python.org/3/library/re.html#module-re) module, but offers additional functionality and a more thorough Unicode support.

(didn't realize they recommend using an external package for regex related stuffs)